scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/rmproxies/RMProxyActiveObject.java

```diff
@@ -179,8 +183,14 @@ public synchronized void cleanCallBack(Future<ScriptResult<?>> future) {
             Entry<Node, ScriptResult<?>> entry = iterator.next();
             if (!PAFuture.isAwaited(entry.getValue())) { // !awaited = arrived
                 if (logger.isInfoEnabled()) {
-                    logger.info("Cleaning script successfull, node freed : " +
-                        entry.getKey().getNodeInformation().getURL());
+                    String nodeUrl = entry.getKey().getNodeInformation().getURL();
+                    logger.info("Cleaning script successfull, node freed : " + nodeUrl
+                        );
+                    try {
+                        logger.info("Cleaning script output on node " + nodeUrl + ":"+future.get().getOutput());
+                    } catch (Exception e) {
```

Comment:
in order to make this cleaner and more robust, I would suggest to put the try/catch statement outside the if(logger.isInfoEnabled()). You can do it this way:
ScriptResult res;
try {
   res = future.get()
} catch (Exception e) {
 logger.warn("blabla")
}